src/sentry/integrations/opsgenie/integration.py

```diff
@@ -165,6 +167,14 @@ def update_organization_config(self, data: MutableMapping[str, Any]) -> None:
             team["id"] = str(self.org_integration.id) + "-" + team["team"]
         return super().update_organization_config(data)
 
+    def migrate_alert_rules(self):
```

Comment:
do you need a separate function to call `apply_async` on a function with the same name? you could also pass the kwargs into the async function directly